spring-integration-core/src/main/java/org/springframework/integration/dsl/TransformerEndpointSpec.java

```diff
@@ -140,7 +140,7 @@ public <P, T> TransformerEndpointSpec transformer(GenericTransformer<P, T> trans
 	 * @param <P> the type ot expect.
 	 * @return the spec.
 	 */
-	public <P> TransformerEndpointSpec expectedType(@Nullable Class<P> expectedType) {
+	public <P> TransformerEndpointSpec expectedType(Class<P> expectedType) {
```

Comment:
Question:
Is this a breaking change if we state that `expectedType` can not be reset to null.  If so should we have a note about it?